tests/src/integration/zipaction/actions/padding/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "my-action",
+  "main": "index.js",
```

Comment:
@houshengbo , I reply all your comments here: this is a sample use case from the openwhisk zip action doc, for zipped actions, yes, that's necessary, you can try by running npm install under the action.